kura/org.eclipse.kura.linux.net/src/main/java/org/eclipse/kura/linux/net/util/KuraConstants.java

```diff
@@ -25,6 +25,7 @@
     Reliagate_10_11("yocto", "1.2.1", "reliagate-10-11"),
     Reliagate_20_25("yocto", "1.2.1", "reliagate-20-25"),
     BoltGATE_20_25("yocto", "1.2.1", "boltgate-20-25"),
+    BoltMAR_20_28("yocto", "1.2.1", "boltmar-20-28"),
```

Comment:
Doesn't this change implies that the exported package in manifest needs to be updated?

Reply:
Even more so, this doesn't look like it's even used in the Kura codebase, or is it? It would be great to not include (or at least try to stop adding new 😉 ) vendor-specific constants in the Kura OSS codebase.